python/setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, Extension
+
+
+with open("../README.md") as f:
+    long_description = f.read()
+
+module = Extension(
+    "miniball",
+    sources=["miniball_python.cpp"],
+)
+
+setup(
+    name="miniball",
+    version="1.0",
+    description="Library to find the smallest enclosing ball of points",
+    author="Martin Kutz",
+    author_email="kutz@math.fu-berlin.de",
```

Comment:
Can you change the author to Kaspar Fischer, Bernd Gärtner, and Martin Kutz; as for the email, Martin Kutz is sadly not with us anymore so I suggest to remove it (the URL points to this GitHub project, which should be a good way to get help).

Reply:
Done!